readme.md

```diff
@@ -70,7 +70,13 @@ __Features__:
         + Many uses including the current state, the recent state, and more
     + TriggerReader class
         + A class that represents a gamepad trigger
-        + Includes simlular state - changing methods like GamepadButton
+<<<<<<< HEAD
```

Comment:
Oops there's a merge conflict here apparently

Reply:
What would cause that?

Reply:
probably if the test branch had old stuff that wasn't pulled before pushing